docs/src/main/sphinx/release/release-444.md

```diff
@@ -0,0 +1,102 @@
+* Fix failure for queries containing large numbers of `LIKE` terms in boolean
+  expressions. ({issue}`21235`)
+
+## Docker image
+
+* Update Java runtime to Java 22. ({issue}`21161`)
+
+## BigQuery connector
+
+* Fix failure when reading BigQuery views with [Apache
+  Arrow](https://arrow.apache.org/docs/). ({issue}`21337`)
+
+## ClickHouse connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Delta Lake connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned access control lists with the native S3 file system. ({issue}`21176`)
+* Add concurrent write reconciliation for non-blind `INSERT`. ({issue}`20983`)
+* Add support for reading tables with [v2
+  checkpoints](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#v2-spec).
+  ({issue}`19345`)
+* Add support for reading [shallow cloned
+  tables](https://docs.databricks.com/sql/language-manual/delta-clone.html).
+  ({issue}`17011`)
+* {{breaking}} Remove support for split size configuration with the catalog
+  properties `delta.max-initial-splits` and `delta.max-initial-split-size`, and
+  the catalog session property `max_initial_split_size`. ({issue}`21320`)
+* Ensure read consistency while performing write operations. ({issue}`21324`)
+
+## Druid connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Hive connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned access control lists with the native S3 file system. ({issue}`21176`)
+
+## Hudi connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned access control lists with the native S3 file system. ({issue}`21176`)
+
+## Iceberg connector
+
+* Add support for the `metadata_log_entries` system table. ({issue}`20410`)
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned access control lists with the native S3 file system. ({issue}`21176`)
+
+## Ignite connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## MariaDB connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## MySQL connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Oracle connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## PostgreSQL connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Redshift connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## SingleStore connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Snowflake connector
+
+* Add support for creating tables with table comments. ({issue}`21305`)
+* Add support for commenting on the existing tables. ({issue}`21305`)
```

Comment:
I would lump these two into into a single entry: "Add support to table comments"